packages/eslint-config-vivy/index.js

```diff
@@ -31,4 +31,14 @@ module.exports = {
       jsx: true,
     },
   },
+  overrides: [
+    {
+      files: ["**/*.ts", "**/*.tsx"],
+      rules: {
+        "no-unused-vars": "off",
```

Comment:
Can we use something like this: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md

As it would be a big loss to not have legit unused vars flag up

Reply:
I only disable it for TS files because it's covered by TS anyway and it conflicts with types (false positives)